From 27fcf78b50546170b2b5e82c906160af7983f809 Mon Sep 17 00:00:00 2001
From: Sampath Vangaveti <sampathv@codeaurora.org>
Date: Sat, 18 Nov 2017 18:19:12 +0530
Subject: [PATCH 14/19] Camera2Client: Disable ZSL by default in QTI
 camera2client

Disable ZSL by default, in QTI specific camera2client.
Use zsl parameter to set zsl on/off if needed.

Change-Id: I07730c865da1e88d4821c14b75f172f84e719c06
---
 .../camera/libcameraservice/api1/qticlient2/Parameters.cpp    | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/services/camera/libcameraservice/api1/qticlient2/Parameters.cpp b/services/camera/libcameraservice/api1/qticlient2/Parameters.cpp
index 41ad7d49e..7f8dd2141 100644
--- a/services/camera/libcameraservice/api1/qticlient2/Parameters.cpp
+++ b/services/camera/libcameraservice/api1/qticlient2/Parameters.cpp
@@ -957,7 +957,7 @@ status_t Parameters::initialize(const CameraMetadata *info, int deviceVersion,
     }
 
     if (isDeviceZslSupported || slowJpegMode ||
-            property_get_bool("camera.disable_zsl_mode", false)) {
+            property_get_bool("camera.disable_zsl_mode", true)) {
         ALOGI("Camera %d: Disabling ZSL mode", cameraId);
         allowZslMode = false;
     } else {
@@ -2010,7 +2010,7 @@ status_t Parameters::set(const String8& paramString) {
         slowJpegMode = true;
     }
     if (isDeviceZslSupported || slowJpegMode ||
-            property_get_bool("camera.disable_zsl_mode", false)) {
+            property_get_bool("camera.disable_zsl_mode", true)) {
         allowZslMode = false;
     } else {
         allowZslMode = isZslReprocessPresent;
-- 
2.17.1

